src/detect-engine-hsbd.c

```diff
@@ -3631,6 +3709,1227 @@ static int DetectEngineHttpServerBodyFileDataTest03(void)
     return result;
 }
 
+/*
+ * Tests buffer inspection like in ips mode,
+ * here all chunks size are smaller than the window_size.
+
+ * in:               buffer:
+ * [ab]              [ab]
+ * [cd]              [abcd]
+ * [ef]              [cdef]
+*/
+static int DetectEngineHttpServerBodyFileDataTest04(void)
+{
+
+    char input[] = "\
+%YAML 1.1\n\
+---\n\
+libhtp:\n\
+\n\
+  default-config:\n\
+\n\
+    http-body-inline: yes\n\
+    response-body-minimal-inspect-size: 1\n\
```

Comment:
the value is loaded from yaml:
Test DetectEngineHttpServerBodyFileDataTest04                     : [31276] 1/6/2015 -- 13:56:29 - (app-layer-htp.c:2235) <Info> (HTPConfigSetDefaultsPhase2) -- 'default' server has 'request-body-minimal-inspect-size' set to 34116 and 'request-body-inspect-window' set to 3973 after randomization.